inst/tinytest/test-lfmcmc.R

```diff
@@ -102,8 +105,19 @@ expected_params_mean <- c(0.3133401, 0.2749686)
 expect_equal(get_mean_params(lfmcmc_model), expected_params_mean, tolerance = 0.0001)
 expect_equal(get_n_params(lfmcmc_model), length(expected_params_mean))
 
-expect_equal(dim(get_accepted_params(lfmcmc_model)), c(n_samp, length(expected_params_mean)))
-expect_equal(dim(get_sample_stats(lfmcmc_model)), c(n_samp, length(expected_stats_mean)))
+expect_equal(get_current_proposed_params(lfmcmc_model), c(0.2699802, 0.1585232), tolerance = 0.00001)
+expect_equal(get_current_accepted_params(lfmcmc_model), c(0.3618486, 0.2823181), tolerance = 0.00001)
```

Comment:
This will probably fail at some point. Let's just check for length, for the moment.